README.md

```diff
@@ -19,13 +19,13 @@ We will be frequently and heavily restructuring code APIs and data chains until
 
 **We are currently only supporting Linux at this time**. You may or may not be able to successfully build and run Holochain on macOS. You definitely won't be able to on Windows (unless you are using WSL, but even that is untested). We will definitely be rolling out support for these OSes in the future, but in the meantime please use Linux for development!
 
-## To Build Binaries
+## Making the Holochain binaries available in your shell
 
```

Comment:
There are a number of contexts and purposes you might be running in which yield different ways to access binaries.

### Building with cargo if you already have rust installed:
```
cargo install --path crates/holochain
cargo install --path crates/dna_util/
```